examples/signTx.py

```diff
@@ -91,6 +96,8 @@ def parse_bip32_path(path):
 )
 
 encodedTx = encode(tx, UnsignedTransaction)
+# encodedTx = bytearray.fromhex(
```

Comment:
Wish there was a comment here saying that this the EIP-1559
(I am pretty sure this is the EIP-1559!)

Reply:
Indeed this was a test for an EIP1559 tx (similar to the test for an EIP2930 tx at line 102). Feel free to submit a PR to add a comment :D @unparalleled-js :)

Reply:
Do you happen to have any advice for encoding a transaction this way in python?
I am struggling with that part

Reply:
I do not :) I probably used web3js or something like that to encode the transaction :)